lib/task-data/tasks/bootstrap-ubuntu.js

```diff
@@ -7,12 +7,12 @@ module.exports = {
     injectableName: 'Task.Linux.Bootstrap.Ubuntu',
     implementsTask: 'Task.Base.Linux.Bootstrap',
     options: {
-        kernelFile: 'vmlinuz-3.16.0-25-generic',
-        initrdFile: 'initrd.img-3.16.0-25-generic',
+        kernelFile: 'vmlinuz-3.19.0-56-generic',
+        initrdFile: 'initrd.img-3.19.0-56-generic',
         kernelUri: '{{ api.server }}/common/{{ options.kernelFile }}',
         initrdUri: '{{ api.server }}/common/{{ options.initrdFile }}',
-        basefs: 'common/base.trusty.3.16.0-25-generic.squashfs.img',
-        overlayfs: 'common/discovery.overlay.cpio.gz',
+        basefs: 'common/base.trusty.3.19.0-56-generic.squashfs.img',
+        overlayfs: 'common/discovery.3.19.0-56-generic.overlay.cpio.gz',
```

Comment:
Yes, my proposal just wants to simplify our future change on this task definition, you can still override all options with your customized values regardless of the `version`. 
Furthermore, maybe the `common/` can also be extracted as a option, such as `baseUrl` or `baseDir` .